src/electron/views/device-connect-view/components/device-connect-port-entry.tsx

```diff
@@ -42,17 +41,26 @@ export class DeviceConnectPortEntry extends React.Component<
         return (
             <div className={styles.deviceConnectPortEntry}>
                 <h3>Android device port number</h3>
-                <MaskedTextField
-                    data-automation-id={deviceConnectPortNumberFieldAutomationId}
-                    ariaLabel="Port number"
-                    onChange={this.onPortTextChanged}
-                    placeholder="Ex: 12345"
-                    className={styles.portNumberField}
-                    maskChar=""
-                    mask="99999"
-                    onKeyDown={this.onEnterKey}
-                />
-                {this.renderValidationPortButton()}
+                <div className={styles.deviceConnectPortEntryBody}>
+                    <MaskedTextField
+                        data-automation-id={deviceConnectPortNumberFieldAutomationId}
+                        ariaLabel="Port number"
+                        onChange={this.onPortTextChanged}
+                        placeholder="Ex: 12345"
+                        className={styles.portNumberField}
+                        maskChar=""
+                        mask="99999"
+                        onKeyDown={this.onEnterKey}
+                        onRenderDescription={() => {
+                            return (
+                                <div className={styles.portNumberFieldDescription}>
+                                    The port number must be between 0 and 99999.
```

Comment:
Can we bring this down to 65535, which is the actual max port number? Ideally, we'd display guidance if someone typed in 80000 as the port number

Reply:
With regards to 65535, I can definitely make that change. I only put 99999 as a user can type that in.

With regards to showing it only when the user types an invalid port number: I discussed this with Fer and we prefer always having the instructions.